executor/envd/service/process_service.py

```
@@ -224,44 +247,72 @@ async def Start(self, request: process_pb2.StartRequest):
+                                                    pty=data
                                                 )
                                             )
-                                    except (OSError, BlockingIOError):
-                                        # No data available or fd closed
-                                        pass
-                                else:
-                                    # Timeout, yield control
+                                        )
+                                    else:
+                                        await asyncio.sleep(0.1)
+                                except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)